Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
     <string name="media_settings_no_files_to_copy">No files to copy</string>
-    <string name="media_settings_copying_file">Copying file %1$d out of %2$d</string>
-    <string name="media_settings_cannot_continue_write_permission">Cannot continue without the WRITE_EXTERNAL_STORAGE permission</string>
-    <string name="media_settings_base_directory_is_already_registered">Cannot use this directory because it is already in use</string>
+    <string name="media_settings_copying_file">Copying file %1$d of %2$d…</string>
+    <string name="media_settings_cannot_continue_write_permission">Grant write permission to external storage first.</string>
+    <string name="media_settings_base_directory_is_already_registered">Cannot use this a folder already in use</string>
 
     <string name="add_dubs_title">Add dubs to post cells</string>
     <string name="add_dubs_description">Appends your GET to your post number, for FUN!</string>
     <string name="files_base_dir_does_not_exist"><![CDATA[Base directory for files does not exist! Set it manually in the settings (Media -> Save location)]]></string>
-    <string name="image_saver_could_not_figure_out_save_location">Couldn\'t figure out save location</string>
-    <string name="settings_captcha_setup">Captcha Settings</string>
-    <string name="settings_captcha_setup_description">Captcha settings are per-site configurable. Tap for site setup.</string>
+    <string name="image_saver_could_not_figure_out_save_location">Could not figure out where to save this</string>
+    <string name="settings_captcha_setup">CAPTCHA Settings</string>
+    <string name="settings_captcha_setup_description">Tap to set up CAPTCHA settings for each site.</string>
     <string name="image_saver_canceled_by_user">Canceled by user</string>
     <string name="image_saver_unknown_location">Unknown location</string>
-    <string name="image_saver_failed_to_save_image">Failed to save the image. Reason: %s</string>
-    <string name="image_saver_no_write_permission">Cannot start saving images without the WRITE_EXTERNAL_STORAGE permission</string>
+    <string name="image_saver_failed_to_save_image">Could not save the image because %s</string>
+    <string name="image_saver_no_write_permission">Grant write permission to external storage to save images</string>
     <string name="media_settings_image_click_preload_strategy_name">Image preload strategy</string>
     <string name="media_settings_image_click_preload_strategy_description">When opening up an image pager, determines what image(s) should be preloaded next.</string>
-    <string name="setting_image_viewer_gestures">Image Viewer gestures</string>
-    <string name="setting_image_viewer_gestures_description">Adds ability to use gestures to close (swipe-up) and save (swipe-bottom) an image/gif/webm when viewing them in gallery</string>
+    <string name="setting_image_viewer_gestures">Image viewer gestures</string>
```

Comment:
"Image Viewer" becomes a proper noun here, capitalise both words.

Reply:
How so? Is it not generic functionality?

Reply:
```suggestion
    <string name="setting_image_viewer_gestures">Image-viewer gestures</string>
```